src/applications/vaos/appointment-list/components/RequestedAppointmentDetailsPage.jsx

```diff
@@ -325,9 +325,9 @@ export default function RequestedAppointmentDetailsPage() {
           <BackLink appointment={appointment} />
           <div className="vads-u-margin-y--2p5">
             <VaAlert status="error" visible>
-              <h2 slot="headline">We couldn’t cancel your request</h2>
+              <h2 slot="headline">We couldn’t cancel your appointment</h2>
               <p>
-                Something went wrong when we tried to cancel this request.
+                Something went wrong when we tried to cancel this appointment.
```

Comment:
Since this component is for request details page, I think it should remain "We couldn’t cancel your request"